packages/dds/tree/src/simple-tree/treeApi.ts

```diff
@@ -82,6 +84,12 @@ export interface TreeNodeApi {
 	 * Returns the {@link TreeStatus} of the given node.
 	 */
 	readonly status: (node: TreeNode) => TreeStatus;
+
+	/**
+	 * Returns the {@link LocalNodeKey} of the given node if the identifier field kind exists.
```

Comment:
We will do more diligence across the identifier docs when the API is more settled (and maybe also remove the concept of "node keys" at that time). Something like this should be okay for now:
```suggestion
	 * If the given node has an identifier specified by a field of kind "identifier" then this returns the compressed form of that identifier.
```

Reply:
Updated with suggestion